<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabOptions</class>
 <widget class="QWidget" name="TabOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabSpacing">
      <number>6</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>790</width>
        <height>343</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Configuration Générale</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_22">
         <property name="text">
          <string>Pseudo :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="pseudo_edit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_23">
         <property name="text">
          <string>Enregistrer dans le dossier :</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="dir_button">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_25">
         <property name="text">
          <string>Nombre de téléchargements simultanés :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spin_max_dwl"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_24">
         <property name="text">
          <string>Nombre de résultats max par page : </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spin_nb_res_page">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="check_clean_dl_list">
         <property name="text">
          <string>Effacer les téléchargements finis au lancement :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="check_icon">
         <property name="text">
          <string>Laisser l'icône dans la zone de notification lorsqu'on ferme Rex :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save_button_1">
           <property name="text">
            <string>Enregistrer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>790</width>
        <height>343</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Configuration Partages</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_7">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_FTP">
         <property name="text">
          <string>Activer le serveur FTP (active la fonction de partage)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>Port :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spin_port">
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>Nombre de connexions simultanées :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spin_connex_sim"/>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="check_share_myFiles">
         <property name="text">
          <string>Partager les fichiers que je télécharge</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="check_aff_maListe">
         <property name="text">
          <string>Permettre l'affichage de ma liste de téléchargements</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <spacer name="horizontalSpacer_10">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save_button_2">
           <property name="text">
            <string>Enregistrer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>790</width>
        <height>343</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Configuration Avancée</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_8">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_30">
         <property name="text">
          <string>Adresse IP du daemon :</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edit_IP_daemon"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_31">
         <property name="text">
          <string>Logs :</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_32">
         <property name="text">
          <string>Nombre d'ip à scanner :</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_34">
         <property name="text">
          <string>Temps entre deux scans en mn :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_35">
         <property name="text">
          <string>Plage d'IPs à scanner :</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edit_plage_ip"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_33">
         <property name="text">
          <string>IPs pouvant configurer le daemon :</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="edit_ip_conf_daemon"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_36">
         <property name="text">
          <string>Afficher ses partages :</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QComboBox" name="combo_aff_myShares">
           <item>
            <property name="text">
             <string>Oui</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Non</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_11">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_37">
         <property name="text">
          <string>Affichage :</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QCheckBox" name="check_expert_mode">
         <property name="text">
          <string>Mode expert</string>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="13" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <spacer name="horizontalSpacer_12">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="save_button_3">
           <property name="text">
            <string>Enregistrer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="combo_logs">
         <item>
          <property name="text">
           <string>Afficher dans la console</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Enregistrer dans un fichier</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spin_nb_ip_scan"/>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="spin_tps_scan">
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>save_button_1</sender>
   <signal>clicked()</signal>
   <receiver>TabOptions</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_button_2</sender>
   <signal>clicked()</signal>
   <receiver>TabOptions</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_button_3</sender>
   <signal>clicked()</signal>
   <receiver>TabOptions</receiver>
   <slot>saveConfig()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>742</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dir_button</sender>
   <signal>clicked()</signal>
   <receiver>TabOptions</receiver>
   <slot>chooseDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveConfig()</slot>
  <slot>chooseDirectory()</slot>
 </slots>
</ui>
